client/components/loadable/loadable_block.re

```diff
@@ -0,0 +1,19 @@
+[@genType]
+[@react.component]
+let make =
+    (
+      ~numLines=1,
+      ~isLoading: bool,
+      ~value: option(React.element)=?,
+      ~children,
+    ) => {
```

Comment:
For better or for worse, prop spreading is not an option in Reason React, so the props here have been implemented a bit differently from the JS version.